LOG.md

```diff
@@ -1,3 +1,13 @@
+# Pulling in TypeORM
+
+Going with postgres (pg).
+Docker compose from [here](https://hub.docker.com/_/postgres).
+
+A few notes:
+
+- Use of rolled back transactions in tests seems to be best practice, but IDs appear to increment in rolled back transactions.
+- We've switched to running all tests sequentially, which will be bad for test performance on devices with a reasonable core count. Eventually we should try to reparallelize these.
```

Comment:
non-blocking: Is it theoretically possible to parallelize tests using the transaction rollback strategy?

Reply:
Maybe! You definitely need to avoid modifying the schemas in parallel, but in theory, I think this should probably be doable.

Not sure!

Filed https://github.com/gpo/grassroots/issues/16